client/components/section-nav/style.scss

```diff
@@ -347,6 +347,17 @@
 	}
 }
 
+.section-nav-tab__title {
```

Comment:
Ooops, sorry, I replied to the wrong comment!

As before, I've tried to match the existing section nav style - the rest of the section styles are in this file too.